docs/source/deployment/databricks/databricks_deployment_workflow.md

```diff
@@ -1,12 +1,12 @@
-# Databricks deployment workflow
+# Developing Kedro projects for Databricks using an IDE
```

Comment:
This title doesn't seem quite right @jmholzer -- did I suggest this and include it in the wrong file? Maybe use "Develop and package a Kedro project for deployment on Databricks" or similar?

Reply:
Would it be possible to do a summary with anchor links to sections like you've done in the other two pages? 

I could add this in if it helps.

Reply:
Hmm yeah it looks like this title crept in in [e0a3094](https://github.com/kedro-org/kedro/pull/2628/commits/e0a3094218510fec12f9661960acb3b17c1f5bda) 🙂 I actually didn't notice it either. I've given this page a better title.

As to the second point I'm a bit confused, as the new page in this PR has a `What this page covers` section, with anchor links as you describe, as do the other two. Have I understood this correctly?

Reply:
You have and I was looking at the wrong version, apologies. Friday afternoon fatigue. Let me review this all properly on Monday morning and approve it then.

Reply:
No problem! Thanks for looking at it late on a Friday :)

Reply:
Oh I already did, though maybe the commit message wasn't clear :)

The new docs pages now have the following titles:
Deploy Kedro projects on Databricks as a job

Reply:
This is great, thanks @jmholzer. When I looked it wasn't there but I've picked up and merged your commit into my final changes now. These were just a few anchors in the sections of the deploy page, and a few style tweaks.

Reply:
Excellent, thanks Jo.

It's a small change, but looking at the titles all together like this, maybe it would be clearer / more consistent to title the pages:

- `Deploy a Kedro project on Databricks as a job`
- `Develop a Kedro project for Databricks using an IDE`
- `Develop a Kedro project for Databricks using the Databricks workspace`

What do you think?